.github/workflows/docker.yml

```diff
@@ -0,0 +1,43 @@
+name: Docker Image Publication
+
+on:
+  push:
+    tags:
+      - java-[0-9]+.[0-9]+.[0-9]+
+
+jobs:
+  publish:
+    runs-on: ubuntu-latest
+
+    defaults:
+      run:
+        working-directory: java
+
+    steps:
+      - name: Set version
+        id: version
+        run: |
+          VERSION=$(echo ${{ github.ref }} | sed -e "s#refs/tags/java-##g")
+          echo ::set-output name=version::${VERSION}
+
+      - name: Login to GitHub Container Registry
+        uses: docker/login-action@v1
+        with:
+          registry: ghcr.io
+          username: ${{ github.repository_owner }}
+          password: ${{ secrets.CR_PAT }}
+
+      - run: ./gradlew docker
+
+      - name: Run Trivy vulnerability scanner
```

Comment:
Should it be checked before creating a tag?

Reply:
Ah, sorry, maybe doing scanning after tagging is better I guess?
I changed the order in https://github.com/scalar-labs/scalar-admin/pull/26/commits/fb90cd172a06795158ee6dcd5fde2348a01c33ee

Reply:
I meant the tagging that triggers this workflow.
If trivy fails, you need to remove the tag and retag it, which is a bit bothering.
So, I thought trivy can be checked before tagging.
But, it is probably difficult with GitHub Actions.